app/styles/about.sass

```diff
@@ -1,36 +1,488 @@
-    font-size: 20px
-    margin-top: 0
+    .nav > li > a
+      // TODO: Move this to bootstrap variables for navbars
+      font-weight: bold
+      font-family: $body-font
+      font-size: 16px
+      padding: 38px 15px 37px
+      color: $burgandy
+      text-shadow: 0 0 0
+
+      &:hover
+        background: $burgandy
+        color: white
+
+    #language-dropdown-wrapper
+      display: inline-block
+      padding: 30px 20px
+      .language-dropdown
+        width: 200px
+
+    .language-dropdown
+      width: 250px
+      
+    @media (max-width: $screen-sm-min)
+      .nav > li > a
+        padding: 10px 20px
+      #language-dropdown-wrapper
+        display: inline-block
+        padding: 10px 10px
+        .language-dropdown
+          width: 150px
+      
+    .img-circle
+      border: $gold 8px solid
+      width: 98px
+      height: 98px // Includes the border
+      
+    .user-level
+      position: absolute
+      top: 76px
+      right: 42px
+      color: $gold
+      text-shadow: 1px 1px black, -1px -1px 0 black, 1px -1px 0 black, -1px 1px 0 black
+      
+  // Buttons
+ 
+  .btn
+    border: none
+    border-radius: 5px
+    font-family: $body-font
+    font-weight: normal
+    background-image: none // overrides legacy buttons
+
+  .btn-primary, .btn-navy
+    background-color: $navy
+    color: white
+  
+  .btn-primary-alt, .btn-navy-alt
+    background-color: white
+    border: 1px solid $navy
+    color: $navy
+    
+  .btn-forest
+    background-color: $forest
+    color: white
+    
+  .btn-forest-alt
+    background-color: white
+    border: 1px solid $forest
+    color: $forest
+    
+  .btn-gold
+    background-color: $gold
+    color: white
+    
+  .btn-gold-alt
+    background-color: white
+    border: 1px solid $gold
+    color: $gold
+  
+  .btn-lg
+    font-size: 18px
+    padding: 13px 35px
+    
+  // Classes
+  
+  .text-navy
+    color: $navy
+    
+  .bg-navy
+    background-color: $navy
+    color: white
+    h1, h2, h3, h4, h5, h6, a
+      color: white
+    a.btn-primary-alt
+      color: $navy
+
+
+#about-view
+  .bold
+    font-weight: bold
+  #nav-container
+    min-height: 55px
+    nav
+      background: $gold
+      &.affix
+        z-index: 1
+        position: fixed
+        top: 0
+        width: 100%
+      ul
+        padding-left: 0
+        list-style: none
+        li
+          height: 22pt
+          display: inline-block
+          margin: 10px 18px 0 18px
+          a
+            color: white
+            text-transform: uppercase
+            text-decoration: none
+        li.active
+          border-bottom: 4px solid white
+    
+  #jumbotron
+    background-color: $navy
+    background-image: url("/images/pages/about/codebackground_zoom_compressed.png")
+    background-size: cover
+    background-repeat: no-repeat
+    // height: 365px
+    
+    h1
+      color: white
+      margin-top: 100px
+      
+    h2
+      color: white
+      margin-bottom: 100px
+      
+  h3, h4
```

Comment:
Ah, in the "view full changes" this makes more sense. FYI github thinks you're referring to code *above* where the comment is inserted, so that's what it shows on the pull request page.